tests/test_registrations/test_views.py

```diff
@@ -492,37 +359,6 @@ def test_get_metaschemas_all(self):
             RegistrationSchema.objects.filter(active=True).count()
         )
 
-    def test_validate_embargo_end_date_too_soon(self):
```

Comment:
Looks like you got rid of the tests for the `validate_embargo_end_date` view, but kept the view itself around in `website/project/views/drafts.py` -- if the view is no longer used and the validation moved to V2, you should delete it too along with these tests.